<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ClipListEditorBase</class>
 <widget class="QWidget" name="ClipListEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MusE: ClipList</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="Q3ListView" name="view" >
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Refs</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Len</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Data</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Clip Properties</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="TextLabel1" >
        <property name="text" >
         <string>Pos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Awl::PosEdit" name="start" />
      </item>
      <item>
       <widget class="QLabel" name="TextLabel2" >
        <property name="text" >
         <string>Len:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Awl::PosEdit" name="len" />
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Awl::PosEdit</class>
   <extends>QWidget</extends>
   <header>awl::posedit.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >awl/posedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
